cw/tvla_cfg_otbn.yaml

```diff
@@ -14,3 +14,5 @@ plot_figures: true
 general_test: true
 mode: otbn
 key_len_bytes: 40
+sample_start: null
+num_samples : 4000
```

Comment:
Could you also add these two lines to `tvla_cfg_template.yaml`, and a comment line `# Only enabled for otbn mode.`?
Eventually, we would like to enable this feature for all modes.

Reply:
yes. good point. Thx for the feedback! I would set both parameters to `null`  as default.